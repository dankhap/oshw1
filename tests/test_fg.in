./infloop.out
@z
jobs
./infloop2.out
@z
jobs
fg 2
quit
