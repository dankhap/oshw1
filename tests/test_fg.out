./infloop.out
@z
./infloop2.out
@z
fg 2
quit

